_data/metadata_registry.yml

```diff
@@ -0,0 +1,28 @@
+# User Metadata
+- name: User Metadata
+  values:
+    - key: avatar
+      description: Avatar that graphical clients can show alongside the user's name
+      format: URL with an optional `{size}` substitution denoting the size to load in pixels
+      examples: https://example.com/avatar/{size}/asdf.jpg
+    - key: bot-url
+      description: >-
+        Lets bot runners provide a URL for more info on a bot user so they can be
+        identified as such in UI
+      format: Valid URL
+      examples: |-
+        https://example.com/bot-info.html
+    - key: display-name
+      description: >-
+        Alternative name to use instead of a nick for display purposes. Useful for gateways
+        to chat services that allow spaces and other characters in nicks. A nick is
+        required for standard protocol level stuff but can be less prominent in the UI
+      format: Any string
+      examples: |-
+        James Wheare
```

Comment:
You can add another example with Cyrillic or Chinese characters.

Reply:
Sounds good, will do